apis/Google.Cloud.Firestore/Google.Cloud.Firestore/FirestoreDb.cs

```diff
@@ -43,11 +43,10 @@ public sealed class FirestoreDb
         /// </summary>
         public string ProjectId { get; }
 
-        // Note: currently internal as only the default database ID is supported.
         /// <summary>
         /// The database ID associated with this database.
```

Comment:
Add comment saying that the value will be "(default)" for the project-default database?